source/rsvp-uk.blade.php

```diff
@@ -0,0 +1,114 @@
+---
+title: RSVP to a Refactoring Discussion
+description: RSVP for an upcoming DevBookClub meeting or discussion.
+---
+@extends('_layouts.master')
+
+@section('body')
+<div class="md-content max-w-6xl mx-auto px-3">
+    <h1 class="mt-8 pb-3 text-4xl border-b-2 border-gray-300">RSVP - Refactoring - UK/EU Group</h1>
+
+    <p class="max-w-3xl mt-10 text-xl">
+        Discussions covering Refactoring: Improving the Design of Existing Code by Martin Fowler are underway. This page is for the group based in the UK. For times that might work better in the US and Canada please see the <a href="/rsvp/">RSVP page for the US-based group</a>.
```

Comment:
I think this text is a little misleading as it might make people think that UK based discussions are already underway. How about something like the following:
"If you would like to join online discussions on Refactoring: Improving the Design of Existing Code by Martin Fowler please use the form below to indicate your interest. For discussion times for North American timezones please see the RSVP page for the US-based group."